integration-tests/integration_tests/integration_tests/end_to_end_tests/int_asynchronous_reliable_messaging_pattern_tests.py

```diff
@@ -71,49 +78,32 @@ def test_should_record_asynchronous_reliable_message_status_as_successful(self):
+            .assertor_for_hl7_xml_message()
+        self._assert_gp_summary_upload_success_detail_is_present(hl7_xml_assertor)
 
         AssertWithRetries(retry_count=10) \
             .assert_condition_met(lambda: DynamoMhsTableStateAssertor.wait_for_inbound_response_processed(message_id))
 
-        DynamoMhsTableStateAssertor(MHS_STATE_TABLE_DYNAMO_WRAPPER.get_all_records_in_table()) \
-            .assert_single_item_exists_with_key(message_id) \
-            .assert_item_contains_values({
-            'INBOUND_STATUS': 'INBOUND_RESPONSE_SUCCESSFULLY_PROCESSED',
-            'OUTBOUND_STATUS': 'OUTBOUND_MESSAGE_ACKD',
-            'WORKFLOW': 'async-reliable'
-        })
+        dynamo_assertor = DynamoMhsTableStateAssertor(MHS_STATE_TABLE_DYNAMO_WRAPPER.get_all_records_in_table())
+        self.assertions.message_status_recorded_as_successfully_processed(dynamo_assertor, message_id)
 
-    def test_should_return_successful_response_from_spine_in_original_post_request_body_if_sync_async_requested(self):
+    def test_should_return_successful_response_and_record_spline_reply_in_resync_table_if_sync_async_requested(self):
         # Arrange
-        message, message_id = build_message('REPC_IN150016UK05', '9446245796')
+        messages = [build_message('REPC_IN150016UK05', '9446245796') for i in range(1)]
 
         # Act
-        response = MhsHttpRequestBuilder() \
-            .with_headers(interaction_id='REPC_IN150016UK05', message_id=message_id, sync_async=True) \
-            .with_body(message) \
-            .execute_post_expecting_success()
-
-        # Assert
-        Hl7XmlResponseAssertor(response.text) \
-            .assert_element_exists_with_value('.//requestSuccessDetail//detail', 'GP Summary upload successful')
-
-    def test_should_record_the_correct_response_between_the_inbound_and_outbound_components_if_sync_async_requested(self):
-        # Arrange
-        message, message_id = build_message('REPC_IN150016UK05', '9446245796')
-
-        # Act
-        MhsHttpRequestBuilder() \
-            .with_headers(interaction_id='REPC_IN150016UK05',
-                          message_id=message_id,
-                          sync_async=True,
-                          correlation_id='1') \
-            .with_body(message) \
-            .execute_post_expecting_success()
+        responses = send_messages_concurrently(messages, interaction_id='REPC_IN150016UK05', sync_async=True)
 
         # Assert
-        DynamoSyncAsyncMhsTableStateAssertor(MHS_SYNC_ASYNC_TABLE_DYNAMO_WRAPPER.get_all_records_in_table()) \
-            .assert_single_item_exists_with_key(message_id) \
-            .assert_element_exists_with_value('.//requestSuccessDetail//detail', 'GP Summary upload successful')
+        all_sync_async_states = MHS_SYNC_ASYNC_TABLE_DYNAMO_WRAPPER.get_all_records_in_table()
+        if has_errors(responses):
+            # TODO: NIAD-72 remove prints and has_errors once sync-async feature is stable
```

Comment:
ToDo left. Is this intended?

Reply:
Intentional and against the JIRA ticket NIAD-72